CrossAppLoginUI/src/main/kotlin/com/infomaniak/core/crossloginui/views/components/CrossLoginSelectAccounts.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Infomaniak Core - Android
+ * Copyright (C) 2025-2025 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.core.crossloginui.views.components
+
+import androidx.compose.foundation.layout.ExperimentalLayoutApi
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.snapshots.SnapshotStateList
+import com.infomaniak.core.crossloginui.components.MultipleAccounts
+import com.infomaniak.core.crossloginui.components.SelectedAccountsButton
+import com.infomaniak.core.crossloginui.components.SingleAccount
+import com.infomaniak.core.crossloginui.data.CrossLoginColors
+import com.infomaniak.core.crossloginui.data.CrossLoginUiAccount
+
+@OptIn(ExperimentalLayoutApi::class)
+@Composable
+internal fun CrossLoginSelectAccounts(
+    accounts: () -> SnapshotStateList<CrossLoginUiAccount>,
+    colors: CrossLoginColors,
+    onClick: () -> Unit,
```

Comment:
Same one as before, this one will be accessed directly in jetpack compose apps

```suggestion
    accounts: () -> SnapshotStateList<CrossLoginUiAccount>,
    onClick: () -> Unit,
    colors: CrossLoginColors = CrossLoginDefaults.colors(),
```